### ISSUE TITLE
Cards freeze in reviewer

### ISSUE THREAD

**Issue by galantra** [Quip13/Anki-Zoom#2]
Hello,

I noticed that after installing _Anki Zoom_, I can't proceed in the reviewer after I have reviewed or suspended a card. The reviewer freezes instead.

I verified that the cause is _Anki Zoom_ by diabling it.

Here you see my other active add-ons:
    
```
    Anki 2.1.20 (47a1bf8b) Python 3.8.0 Qt 5.14.1 PyQt 5.14.1
    Platform: Windows 10
    Flags: frz=True ao=True sv=1
    Add-ons, last update check: 2020-03-03 07:42:24
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['1135507716 tagedit_subtag_completer', 0, 'None', '']
    '' ['autoLearningSteps', 0, 'None', '']
    '' ['rememorize_btn', 0, 'None', mod]
    '' ['show_warning_if_no_tags_when_adding_cards', 0, 'None', '']
    '' ['touch_wood', 0, 'None', '']
    '' ['visual_feedback', 0, 'None', '']
    3 add-ons merged quicker anki explain deletion explain database check ['777545149', 2019-09-28T08:05, 'None', '']
    3ft Under Bury Newly Added Cards ['32017685', 2019-08-19T21:16, 'None', mod]
    Add Hyperlink ['318752047', 2019-10-21T09:17, 'None', '']
    Add Parent Tag To Every Tag In A Deck ['543106186', 2019-11-21T13:20, 'None', '']
    Add Table ['1237621971', 2019-11-08T21:17, 'None', mod]
    Advanced Browser ['874215009', 2020-02-21T10:49, 'None', mod]
    Advanced Copy Fields ['1898445115', 2019-08-17T11:52, 'None', '']
    Agent Orange Pseudo-Fuzz Defoliator ['175851166', 2019-08-13T09:25, 'None', '']
    AnkiConnect ['2055492159', 2020-01-06T18:26, 'None', '']
    Answer Button Sounds ['679615590', 2020-02-23T07:30, 'None', '']
    Batch Editing ['291119185', 2019-08-08T19:56, 'None', '']
    Blitzkrieg II - Advanced Browser Sidebar BETA7 ['564851917', 2020-02-26T08:52, 'None', '']
    Bluetooth Headset Audio Lag Fix ['880422015', 2019-12-16T09:03, 'None', '']
    Browser TableEditor side-by-side horizontal split ['831846358', 2020-01-14T00:07, 'None', mod]
    Bury related notes ['413416269', 2019-08-08T21:27, 'None', mod]
    Butler-Addon ['1439937507', 2020-02-23T07:12, 'None', '']
    CTRLF5 to Refresh the Browser ['1347728560', 2019-08-08T20:58, 'None', '']
    Cacher In The Rye Windows ['1302452246', 2020-01-16T18:48, 'None', '']
    Cardistry Dynamically Adjust New Cards ['1535078906', 2020-02-16T10:48, 'None', '']
    Change subdecks configuration ['751420631', 2019-11-17T08:01, 'None', '']
    Close and lose current input  for sticky fields ['1284221829', 2019-10-16T09:07, 'None', '']
    Cloze Overlapper ['cloze_overlapper', 2019-02-01T13:37, 'None', '']
    Copy notes ['1566928056', 2019-11-17T07:23, 'None', mod]
    Correct Answers Graph ['993120073', 2020-01-06T18:26, 'None', '']
    Correcting a bug in anki which makes new card appearing in wrong order ['127334978', 2019-12-09T08:43, 'None', '']
    CrowdAnki JSON exportimport Edit history Collaborate on deck creation ['1788670778', 2020-02-10T16:24, 'None', '']
    Customize Keyboard Shortcuts ['24411424', 2020-03-03T07:28, 'None', mod]
    Deck name in title 21 ['699175524', 2019-08-23T23:06, 'None', '']
    Delete empty NEW cards tag other empty cards ['1402327111', 2020-02-26T08:52, 'None', '']
    Do not underline current queue in reviewer ['267531099', 2019-08-17T12:23, 'None', '']
    Dont remove mark on export ['909480379', 2020-02-16T10:53, 'None', '']
    Ease Factor Histogram ['494944895', 2019-08-07T22:57, 'None', '']
    Empty cards returns more usable informations ['25425599', 2019-12-07T17:03, 'None', '']
    Enhance main window ['877182321', 2020-02-16T10:48, 'None', mod]
    Ensure each note is unique correct a bug introduced by add-on copy note ['2082040683', 2019-09-28T08:09, 'None', '']
    Extended Schema Hash ['1187737890', 2020-01-06T18:25, 'None', '']
    Extended Tag AddEdit Dialog ['1135507717', 2020-02-19T22:09, 'None', '']
    Forked Improved Quzlet to Anki Importer english ['1302146328', 2019-11-24T07:56, 'None', '']
    Frozen Fields ['516643804', 2020-03-01T17:18, 'None', '']
    HUMBLE PIE distinct cards studied today statistic ['2133933791', 2020-03-02T09:28, 'None', '']
    High Yield Tags ['1918380616', 2020-02-14T09:45, 'None', mod]
    Highlight Search Results in the Browser ['225180905', 2019-08-15T18:52, 'None', '']
    Hoarders Delight RecycleTrash Bin ['1758624338', 2019-08-13T10:59, 'None', '']
    Holdem Cardfield Preserves add card history ['363320830', 2019-08-13T10:23, 'None', '']
    HoochieBaby Queue Controller  Day-Lrn Q ['1847358755', 2019-08-09T10:00, 'None', '']
    HoochieMama Randomize Rev Queue ['1460733408', 2020-02-16T10:48, 'None', '']
    HoochiePapa Randomize New Queue ['1173108619', 2019-08-09T10:00, 'None', '']
    If a note has no more card warns instead of deleting it ['2018640062', 2019-08-13T10:19, 'None', '']
    Ignore accents in browser search ['1924690148', 2019-08-07T22:32, 'None', '']
    Image Occlusion Enhanced for Anki 21 alpha ['1374772155', 2019-09-11T14:53, 'None', '']
    ImageResizer ['1214357311', 2019-08-07T23:43, 'None', '']
    Improve speed of change of note type ['115825506', 2019-11-17T08:01, 'None', '']
    Improved Quizlet to Anki 21 Importer ['538351043', 2020-02-26T08:52, 'None', '']
    Improving change note type ['513858554', 2019-08-10T12:57, 'None', '']
    Increase autosave frequency ['786971082', 2019-08-13T18:26, 'None', mod]
    Keep card displayed in reviewer until it is modified ['1894745652', 2020-01-25T09:49, 'None', '']
    Keep files git svn in add-on folders ['225953877', 2019-12-07T17:00, 'None', '']
    Large Fancy Buttons compatible with ReMemorize Buttons ['1042429613', 2019-10-28T10:20, 'None', '']
    Load Balanced Scheduler ['208879074', 2019-08-08T20:32, 'None', '']
    Long term backups ['529955533', 2019-08-08T20:16, 'None', '']
    Markdown Support - With Batch n Preview ['1259478414', 2019-11-15T19:18, 'None', mod]
    Mini Format Pack ['295889520', 2019-08-07T21:49, 'None', mod]
    More Decks Stats and Time Left ['1556734708', 2020-01-22T11:43, 'None', '']
    Newline in strings in add-ons configurations ['112201952', 2019-09-28T08:09, 'None', '']
    Night Mode ['z1496166067', 2020-01-29T22:27, 'None', '']
    No Distractions Full Screen Clean review interface  TabletTouch support ['1049863218', 2020-03-01T17:18, 'None', mod]
    OpenInExternalEditorRenameDuplicate for ImageAudioVideo ['1560623518', 2019-12-08T17:15, 'None', mod]
    Opening the same window multiple time ['354407385', 2019-08-08T20:35, 'None', '']
    Polar Connect - PDF and Incremental Reading with Anki Sync ['734898866', 2019-08-07T22:36, 'None', '']
    Preview any cloze number ['915063177', 2020-02-29T07:30, 'None', '']
    Progress Graphs and Stats for Learned and Matured Cards ['266436365', 2019-08-14T07:50, 'None', '']
    Push Paradox Sibling Cardistry ['1797395052', 2019-08-13T10:51, 'None', '']
    ReMemorize Ex Rated0Search ['1056004913', 2019-08-09T09:34, 'None', '']
    ReMemorize Rescheduler with sibling and logging v130 ['323586997', 2020-02-04T17:14, 'None', mod]
    Reloadbackup menu option ['883617253', 2020-02-10T19:06, 'None', '']
    Removes Empty Note Types for 21 ['530953383', 2019-08-17T08:30, 'None', '']
    Reviewer to Browser choosing what to display ['1555020859', 2019-08-13T07:25, 'None', '']
    Search and Replace Tags ['138501288', 2019-09-12T10:23, 'None', '']
    Searching PDF Reading  Note-Taking in Add Dialog ['1781298089', 2020-03-02T07:20, 'None', mod]
    Slackers Delight PostponeDefer Button ['880824023', 2020-02-27T20:59, 'None', '']
    Statistics for Note Types Card Types and Tags ['16703369', 2019-12-17T09:22, 'None', '']
    Straight Reward Ease Factor grows at 5 straight Good rating in review ['1130426680', 2019-08-07T22:57, 'None', '']
    SyncRedirector ['2124817646', 2019-12-13T06:56, 'None', '']
    Tag Entry Enhancements ['1348430474', 2019-09-13T15:32, 'None', '']
    True Retention by Card Maturity ['923360400', 2019-08-10T10:00, 'None', '']
    add-on dialog searchfilter bar ['561945101', 2020-01-22T11:43, 'None', '']
    avgEase ['633293039', 2019-08-07T23:01, 'None', '']
    bury future new sibs ['930365039', 2019-11-25T09:52, 'None', mod]
    copy note contents to new Add window from Browser Reviewer Add EditCurrent ['759018284', 2019-12-09T08:43, 'None', mod]
    editor apply font color background color custom class custom style ['1899278645', 2020-01-23T09:14, 'None', '']
    extended table editor ['805891399', 2020-02-09T17:57, 'None', mod]
    extended template editor with save and diff also for html source of fields ['1043915942', 2019-11-25T12:13, 'None', mod]
    remove linebreaks  do word wrap fork for 21 ['2028179508', 2019-08-18T09:48, 'None', mod]
    
    ===IDs of active AnkiWeb add-ons===
    1042429613 1043915942 1049863218 1056004913 112201952 1130426680 1135507717 115825506 1173108619 1187737890 1214357311 1237621971 1259478414 127334978 1284221829 1302146328 1302452246 1347728560 1348430474 1374772155 138501288 1402327111 1439937507 1460733408 1535078906 1555020859 1556734708 1560623518 1566928056 16703369 175851166 1758624338 1781298089 1788670778 1797395052 1847358755 1894745652 1898445115 1899278645 1918380616 1924690148 2018640062 2028179508 2055492159 2082040683 208879074 2124817646 2133933791 225180905 225953877 24411424 25425599 266436365 267531099 291119185 295889520 318752047 32017685 323586997 354407385 363320830 413416269 494944895 513858554 516643804 529955533 530953383 538351043 543106186 561945101 564851917 633293039 679615590 699175524 734898866 751420631 759018284 777545149 786971082 805891399 831846358 874215009 877182321 880422015 880824023 883617253 909480379 915063177 923360400 930365039 993120073
```

**Comment by jamiekoi** [comment]
This issue has been fixed in the latest update V1.1.1. Have you updated it?

**Comment by galantra** [comment]
Yes, it's been fixed!
I had not updated because the update is not yet caught by the "Check for updates" function in Anki.
I downloaded it by going to "Get add-ons" and entering the code instead.
